src/options.cpp

```diff
@@ -1476,6 +1476,58 @@ void options_manager::add_options_graphics()
 
     mOptionsSort["graphics"]++;
 
+    add( "FONT_BLENDING", "graphics", translate_marker( "Font blending" ),
+        translate_marker( "If true, fonts will look better." ),
```

Comment:
Please someone tell me what font blending is doing, so I can write a accurate description.

Reply:
Looks like it's making it look much nicer by using alpha values to create a smoother look. https://www.libsdl.org/projects/docs/SDL_ttf/SDL_ttf_35.html